web/src/components/shared/cart/cart-purchase-summary.tsx

```diff
@@ -42,6 +51,28 @@ const Total = styled.div`
 `;
 
 const PurchaseSummary = ({ cart }: { cart: CP[] }) => {
+  const [Error, setError] = useState('');
+  const clickHandler = () => {
+    const paymentDataRequest = {
+      items: cart.map((item) => {
+        return {
+          title: item.name,
```

Comment:
add picture_url to the model of the item and to the lamda api so we can show the picture inside mercadopago.
<img width="972" alt="Screenshot 2020-10-03 at 10 35 07" src="https://user-images.githubusercontent.com/8405459/94987123-23c65100-0564-11eb-83fc-13e2bbd3a2d8.png">